scripts/tax-id-analysis/check-duplicate-tax-ids.py

```diff
@@ -0,0 +1,47 @@
+import json
```

Comment:
This and the .ts file are almost identical. Initially wrote this as a python script for ease of data manipulation. Opted for TS so that I could utilize `AWSEncryptionDecryptionFactory`.